### DISCUSSION TITLE
(O-C) Analysis

### DISCUSSION THREAD

**Discussion by kjkoeller** [kjkoeller/EclipsingBinaries#128] [Ideas]
Use the linear coefficient and add it to the period and re-analyze the (O-C) values with this new period.

**Comment by MartinKjarP6N** [comment]
Sure thing!

To re-analyze the (O-C) values with a new period, you'll need to use the linear coefficient and add it to the original period. This will give you the new, adjusted period which you can use to recalculate the (O-C) values.

To do this, first calculate the linear coefficient using linear regression. Once you have the linear coefficient, you can add it to the original period to get the adjusted period. Then, use this adjusted period to recalculate the (O-C) values.

This process is important because it helps to account for any changes or variations in the observed period over time, which can be caused by a variety of factors such as gravitational interactions or variations in the star's luminosity. By adjusting for these factors and re-analyzing the (O-C) values, you can get a more accurate understanding of the star's true period and behavior.

**Reply by kjkoeller** [reply]
Thanks for the comment and explanation! This should be easy to implement as the program currently outputs the linear regression and quadratic regression from a Least Squares Fit. All that needs to be done is adding this already known value to the period and then recalculating and plotting the (O-C) values.

**Comment by kjkoeller** [comment]
I think I have this done, but haven't tested it all the way through from first calculating to re-calculating Times of Minimum (ToM). I know the new period is calculated correctly, but I am not sure if the usage of the new period is being correctly utilized within the code. At minimum, the code outputs the new period, so the user could relaunch the code and use that new period instead of the one first used.
